dashdotdb/__init__.py

```diff
@@ -27,7 +27,7 @@ class DashDotDb(App):
     def create_app(self):
         # pylint: disable=redefined-outer-name
         app = Flask(self.import_name, **self.server_args)
-
+#       app.config['SQLALCHEMY_ECHO'] = True
```

Comment:
letft over?

Reply:
yep. was thinking: it'd be helpful to add a debug toggle here, but again: more (un-necessary) scope. will clean up.

Reply:
removed. did note in readme how to enable.. because knowing me, i'll probably forget..